services/scan/src/precinct_scanner_app_scanning.test.ts

```diff
@@ -545,16 +525,14 @@ test('invalid ballot rejected', async () => {
   await waitForStatus(app, {
     state: 'rejecting',
     interpretation,
-    canUnconfigure: false,
   });
   await waitForStatus(app, {
     state: 'rejected',
     interpretation,
-    canUnconfigure: false,
   });
 
   await mockPlustek.simulateRemoveSheet();
-  await waitForStatus(app, { state: 'no_paper', canUnconfigure: false });
```

Comment:
Previously the behavior was that you couldn't unconfigure after any batch had been started, even if the batch was empty or only contained rejected (deleted) sheets. Since now, the batch starts with polls open, that means that you could not unconfigure after polls open. Behavior changed to that you can unconfigure if there are no counted (accepted) ballots.